horovod/keras/__init__.py

```diff
@@ -144,3 +144,55 @@ def broadcast(value, root_rank, name=None):
     """
     bcast_op = hvd.broadcast(tf.constant(value, name=name), root_rank)
     return K.get_session().run(bcast_op)
+
+
+def load_model(filepath, custom_optimizers=None, custom_objects=None,
+               compile=True):
+    """
+    Loads a saved Keras model with a Horovod DistributedOptimizer.
+
+    The DistributedOptimizer will wrap the underlying optimizer used to train
+    the saved model, so that the optimizer state (params and weights) will
+    be picked up for retraining.
+
+    By default, all optimizers in the module `keras.optimizers` will be loaded
+    and wrapped without needing to specify any `custom_optimizers` or
+    `custom_objects`.
+
+    # Arguments
+        filepath: One of the following:
+            - string, path to the saved model, or
+            - h5py.File object from which to load the model
+        custom_optimizers: Optional list of Optimizer subclasses to support
+            during loading.
+        custom_objects: Optional dictionary mapping names (strings) to custom
+            classes or functions to be considered during deserialization.
+        compile: Boolean, whether to compile the model after loading.
```

Comment:
Mention compile is also optional and defaults to True.

Reply:
This param is not available in Keras 2.0.0, so removed.